src/launch/cli/service/create/__init__.py

```diff
@@ -213,7 +213,7 @@ def create(
         repository=repository,
         input_data=input_data,
         git_message=git_message,
-        uuid=not skip_uuid,
+        uuid=skip_uuid,
```

Comment:
Seems like a strange change to make in isolation. Are we fixing the code to match the behavior laid out in the documentation?